docs/src/folderstruct.md

```diff
@@ -0,0 +1,55 @@
+Note: the documentation can have sub-menu. For example, `APIs` have seb-menu `ClimaLSM` which has many sub-menu... This structure is built in our current framework via a file `docs/list_of_apis.jl` in that example, which is then included in `docs/make.jl`.   
+
+## /experiments folder
+
+The `experiments` folder contains scripts to run ClimaLSM models. It contains a folder for `integrated` models and a folder for `standalone` models. It is meant to provide users with simple examples of ClimaLSM runs. The files contains meteorological inputs (such as precipitation), values for every parameters, and the domains and timestepper are specified. 
+
+For example, `/experiments/LSM/ozark/` contains:
+- ozark_domains.jl: Describes the soil domain (depth, number of layer), and the canopy (number and height of stems and leaves).
+- ozark_met_drivers_FLUXNET.jl: This files load meteorological input data from the ozark FLUXNET file, and does additional things such as spline interpolation of these drivers.
+- ozark_parameters.jl: In this file, parameters values are defined, for example the soil porosity is given a value of 0.55 m3 m-3. 
+- ozark_simulation.jl: In this file, initial and final time are set, as well as time resolution and time stepper algorithm. 
+- ozark.jl: running this script will include all the above scripts, and run ClimaLSM for the single-site ozark. It will produce output in a text file as well as some figures comparing data and simulation.
+
+## /src folder
+
+The `/src` folder contains the source code of `ClimaLSM` models. It contains 3 folders:
+- SharedUtilities: This is a core folder that defines functions to create `ClimaLSM` models, and many utility functions to be called across the repository. For example, `SharedUtilities/models.jl` defines and export the function `make_update_aux` which will be used to store auxiliary variables in a nested variable `p`, `SharedUtilities/boundary_conditions.jl` defines functions for setting boundary condition for ODE domains, etc.
+- Standalone: This folder contains standalone models, which are submodels that can be run independently of each other. This is an important aspect of `ClimaLSM` code design: to maximise modularity, sub-models can be run alone, and many different methods of the same sub-model can be defined via Julia multiple-disptach. 
+- Integrated: This folder contains integrated models. It assembles standalone models together, as one would assemble pieces of a puzzle. Thanks to the modularity of `ClimaLSM` design, many configuration of LSM can be assembled in integrated models. 
+
+As well as 3 files:
+- ClimaLSM.jl: This file is the main Julia module of `ClimaLSM.jl` repository. It contains all functions defined in `/src` in a nested way, for example `ClimaLSM.X`, `ClimaLSM.Soil.X`, 'ClimaLSM.Canopy.X`, etc. When a Julia user install and uses ClimaLSM via `]add ClimaLSM, using ClimaLSM`, they are loading those functions, and are ready to use ClimaLSM codebase. 
+- Parameters.jl: 
```

Comment:
This is how we interact with CLIMAParameters, which describes a single interface and parameter set that all Clima models use. This ensures all components of the Earth System Model use the same values for all parameters, and that the same parameter calibration tools can be used to calibrate all models, either jointly or independently.